### PR TITLE
Footer Web and Hero Section

### PR DESCRIPTION
**Pull request by Reshex**
Changed footer component and scss, also added a new component for the hero section and added scss.
I haven't separated the pull request because i was waiting for a reply regarding the background and the font, and didn't want to wait in order to continue to the next task.
In the future ill make a separated branches for each section or feature.

### CODE REVIEW

File: src/components/Pages/HomeSections/footer/footerStyle.scss [hunk 1]
```diff
@@ -1,11 +1,12 @@
+$BackgroundColor: #81a0d3;
 .footer {
     width: 100%;
     height: fit-content;
     display: flex;
     flex-direction: column;
     align-items: center;
     position: relative;
-    background-color: rgb(117, 169, 228);
+    background-color: $BackgroundColor;
```
Comment (vladbyk89) [inline src/components/Pages/HomeSections/footer/footerStyle.scss]: General colors that will be later reused in other CSS files should be defined in globals.scss

**Review by talyaron — CHANGES_REQUESTED** [review]
Hi Bar,

Please look at the footer in mobile mode, and you will be able to see that it is not mobile frendly. please consult with Mash.

**Review by talyaron — CHANGES_REQUESTED** [review]
Also, the Hero Section is not responsive. 
and please look at the image:
 <Image style={{ width: "38rem", height: "35rem" }} className="heroSectionImg" src={heroSectionImg} alt="Hero Section" />
the constant style should be only on the scss.

Thanks

**Review by talyaron — CHANGES_REQUESTED** [review]
Hi bar, good work.
I need two changes:
- make the scss a module (css module). 
- about the varibales, please put them on the global.scss, with this method: --variable...

**Comment by vladbyk89** [comment]
Good job so far. I have a few comments.
1) Make sure the text aligns the way it is presented in figma.
2) The bottom part, where the arrow is, could be shorter. (compare with figma design)

**Comment by Reshex** [comment]
> Good job so far. I have a few comments.
> 
> 1. Make sure the text aligns the way it is presented in figma.
> 2. The bottom part, where the arrow is, could be shorter. (compare with figma design)

Hey Vladi, thank you for the feedback, I believe that you were referring only to the footer, I fixed the problems and also fixed some problems in the hero section component.
As i said before, I accidently made a mistake that both of the components are at the same branch, for the next task I will use another branch.

**Review by talyaron — APPROVED** [review]
Well done

**Pull request merged**